playground/nextjs/pages/organization/[[...index]].tsx

```diff
@@ -7,7 +7,7 @@ const OrganizationProfilePage: NextPage = () => {
     <div
       style={{ display: 'flex', flexDirection: 'column', gap: '2rem', justifyContent: 'center', alignItems: 'center' }}
     >
-      <OrganizationProfile />
+      <OrganizationProfile path='/organization' />
```

Comment:
❓ Will this error on dev time due to missing type?

Reply:
This error will be shown when the component is being rendered. This is a runtime error